examples/linear_load.jmx

```diff
@@ -75,16 +73,6 @@ to let Throughput Shaping Timer limit the load and create requested load. </stri
         <BackendListener guiclass="BackendListenerGui" testclass="BackendListener" testname="Backend Listener" enabled="false">
```

Comment:
I've just checked, the listener is `enabled="false"` so this test will work without InfluxDB, no need to remove this configuration. Sorry, my bad. You can roll it back.

Reply:
It's fine, I actually removed this even before. I really think that our examples files should not cover that at all